src/sql/parts/notebook/models/notebookModel.ts

```diff
@@ -163,6 +171,10 @@ export class NotebookModel extends Disposable implements INotebookModel {
 		return this._contextsChangedEmitter.event;
 	}
 
+	public get contextsChanging(): Event<void> {
```

Comment:
Can we put the word 'loading' somewhere here? Based on our conversation, sounds like this is used to put one (or both) dropdown in a loading state.